### PR TITLE
Docker composer and necessary scripts for running lighthouse

### PR DESCRIPTION
**Pull request by atif-konasl**
Prepared docker compose file with necessary scripts. 
Need to incorporate `lighthouse-docker-test` directory into current directory structure.

### CODE REVIEW

**Review by NandoTheessen — CHANGES_REQUESTED** [review]

File: l16beta/lighthouse-docker-test/.env [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## geth related env variables
+GETH_HTTP_PORT=8545
+GETH_PEER_PORT=30303
+GETH_VERBOSITY=3
+GETH_VERSION="v0.2.0-dev"
+
+
+## prysm related env variables
+PRYSM_BEACON_VERSION="v0.2.2-dev"
```
[the remark this answers is not in the document]
Comment (atif-konasl) [reply l16beta/lighthouse-docker-test/.env:35]: Thanks. Changing it.

File: l16beta/docker/docker-compose.lighthouse.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      - $VALIDATOR_DATA_VOLUME:/validator_data
+      - $CONFIGS_VOLUME:/configs
+      - $LH_SCRIPTS_VOLUME:/scripts
+    command: sh /scripts/lighthouse/create-wallet.sh
+    network_mode: host
+
+  eth2stats-client:
+    image: macht/eth2stats-client:v1.0.0
+    container_name: eth2stats-client
+    depends_on:
```
[the remark this answers is not in the document]
Comment (atif-konasl) [reply l16beta/docker/docker-compose.lighthouse.yml:109]: Thanks. Changing it.

File: l16beta/lighthouse-docker-test/scripts/create-wallet.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
```
[the remark this answers is not in the document]
Comment (atif-konasl) [reply l16beta/lighthouse-docker-test/scripts/create-wallet.sh:1]: I have tried without scripts but i could not manage to do it.
Help is appreciated

File: l16beta/docker/docker-compose.lighthouse.yml [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+      - lighthouse_beacon
+    command: >
+      run
+      --beacon.type=v1
+      --eth2stats.node-name=$NODE_NAME
+      --eth2stats.addr=$ETH_2_STATS_ADDRESS
+      --eth2stats.tls=false
+      --beacon.metrics-addr=http://localhost:5054/metrics
```
[the remark this answers is not in the document]
Comment (atif-konasl) [reply l16beta/docker/docker-compose.lighthouse.yml]: Actually `docker-compose.lighthouse.yml` file was accidentally pushed.
I have already removed that file.

File: l16beta/docker/docker-compose.lighthouse.yml [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+      --metrics
+      --init-slashing-protection
+      --beacon-nodes http://localhost:5052
+      --graffiti $GRAFFITI
+      --suggested-fee-recipient $ETHERBASE
+    network_mode: host
+
+  lighthouse_account:
+    image: europe-docker.pkg.dev/lks-lz-artifacts/docker-lighthouse/beacon:$LH_BEACON_VERSION
+    container_name: lighthouse_validator
```
[the remark this answers is not in the document]
Comment (atif-konasl) [reply l16beta/docker/docker-compose.lighthouse.yml]: Actually `docker-compose.lighthouse.yml` file was accidentally pushed.
I have already removed that file.

**Comment by atif-konasl** [comment]
Closing this draft PR

**Pull request closed without merging**